### PR TITLE
Merge Queue tweaks

### PR DESCRIPTION
**Pull request by Eds-Dbug**
Closes #

a.) Removes top of page pagination.
b.) Removes the "Show requests that I've reviewed" and "Show my requests" at the top of the page.
c.) Makes the make the MR title a link to preview the merge without assigning.

### Technical

Changes in merge_request_table.html only

### Testing

Mostly visual. Will update next section with images.

### Screenshot

Will update

### Stakeholders

@mekarpeles @jimchamp

### CODE REVIEW

File: openlibrary/templates/merge_request_table/table_row.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
         $# End : Status dot indicator
 
         <div class="mr-details">
-        $:_('<span class="mr-details__request-title">%(title)s</span>', title=request_title)
+        $:_('<span class="mr-details__request-title"><a href="%(url)s" target="_blank">%(title)s</a></span>', title=request_title, url=request["url"])
```
Comment (jimchamp) [inline openlibrary/templates/merge_request_table/table_row.html:29]:
```suggestion
        <span class="mr-details__request-title">$request_title</span>
```
Remove link.  Render span itself --- there's nothing to translate, so no need to use `gettext` here.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @Eds-Dbug. I've added some review notes.

Adding a link to the merge page is a little more involved than just nesting the title in an anchor element, for a couple of reasons:
1. When unauthenticated, the author and work merge pages return `404`s and `403`s, respectively.  I didn't check, but I suspect that the same will be true for authenticated patrons who are not members of either the `librarians`, `super-librarians`, or `admin` usergroups.
2. `mrid` is necessary to close merge requests.  If a super-librarian clicks a title link, and then clicks the merge button, the corresponding merge request will not be closed.  Worse still, resubmitting a work or author merge has unknown, but potentially very harmful, effects.

Item 1 is easy enough to fix, but item 2 will likely require some community discussion to come up with a good solution.

Go ahead and remove that link for now.

**Comment by Eds-Dbug** [comment]
Noted ill do the fix and hop on a communal discussion.

**Review by jimchamp — APPROVED** [review]
Lgtm.  Thanks, @Eds-Dbug.

image

**Pull request merged**